run_pylint.sh

```diff
@@ -73,6 +86,17 @@ record_return() {
     fi
 }
 
+check_print() {
+    PRINTS=$(egrep --count '^\s*print\s+' "$1")
```

Comment:
The duplication here bothers me. Why not this?

    PRINTS=$(egrep --with-filename --line-number '^\s*print\s+' "$1")
        if [ -n "$PRINTS" ]
            echo "Python print ..."
            echo "$PRINTS"

I've tested it, with multiple prints, and it works.